src/web/demo.rkt

```diff
@@ -235,7 +235,11 @@
                   #"OK"
                   (current-seconds)
                   APPLICATION/JSON-MIME-TYPE
-                  (list (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*")))
+                  (if (hash-has-key? resp 'job-id)
+                    (list 
+                      (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*"))
+                      (header #"Herbie-job-id" (string->bytes/utf-8 (hash-ref resp 'job-id))))
```

Comment:
Use `X-Herbie-Job-ID`; the `X` indicates that it's a private use header.

Reply:
TIL